docs/orchestrating-elastic-stack-applications/elasticsearch/virtual-memory.asciidoc

```diff
@@ -11,7 +11,7 @@ endif::[]
 By default, Elasticsearch uses memory mapping (`mmap`) to efficiently access indices.
 Usually, default values for virtual address space on Linux distributions are too low for Elasticsearch to work properly, which may result in out-of-memory exceptions. This is why link:k8s-quickstart.html[the quickstart example] disables `mmap` through the `node.store.allow_mmap: false` setting. For production workloads, it is strongly recommended to increase the kernel setting `vm.max_map_count` to `262144` and leave `node.store.allow_mmap` unset.
 
-The kernel setting `vm.max_map_count=262144` can be set on the host either directly or by a dedicated init container, which must be privileged. To add an init container that changes the host kernel setting before your Elasticsearch pod starts, you can use the following example Elasticsearch spec:
+The kernel setting `vm.max_map_count=262144` can be set on the host directly, by a dedicated init container which must be privileged, or a dedicated Daemonset as seen in the <<{p}-autopilot>> section. To add an init container that changes the host kernel setting before your Elasticsearch pod starts, you can use the following example Elasticsearch spec:
```

Comment:
As mentioned in my previous comment, I think I would do this the other way around because it can help on other types of K8S clusters:

## Virtual memory

By default, Elasticsearch uses memory mapping (mmap) to efficiently access indices.....

### Using an init container

The kernel setting vm.max_map_count=262144 can be set on the host directly, by a dedicated init container....

### Using a DaemonSet

The kernel setting can also be set on all the cluster nodes using a `DaemonSet` .... pros/cons ... This is the recommended approach on Google Autopilot<<{p}-autopilot>>.